packages/js/wasm/src/WasmWrapper.ts

```diff
@@ -163,13 +119,17 @@ export class WasmWrapper extends Wrapper {
         },
         invokeResult: {} as InvokeResult,
         method,
-        args: isBuffer(args) ? args : msgpackEncode(args),
-        env: msgpackEncode(this._getClientEnv()),
+        args: args
+          ? isBuffer(args)
+            ? args
+            : msgpackEncode(args)
+          : EMPTY_ENCODED_OBJECT,
```

Comment:
nit: I would convert this to a variable, making this line more readable

Reply:
I find this really readable (though if you're not used to ternary operators I can see how it wouldn't be)
If you have a more readable code snippet pls post it :)